src/EvoSC.Common/Services/MapService.cs

```diff
@@ -87,7 +87,7 @@ public async Task<IMap> AddMapAsync(MapStream mapStream)
         }
 
         await _serverClient.Remote.InsertMapAsync($"EvoSC/{fileName}");
-
+        await _serverClient.Remote.SaveMatchSettingsAsync($"MatchSettings/{_config.Path.DefaultMatchSettings}");
```

Comment:
I strongly recommend to use `IMatchSettingsService` for editing match settings.

Reply:
Already resolved